dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -895,75 +895,58 @@ fn print_task_details(bgtask: &BackgroundTask, details: &serde_json::Value) {
             /// total number of instances checked
             total_instances: usize,
 
-            /// number of instances whose check succeeded without a state
-            /// change
-            no_change: usize,
-
-            /// number of instances whose state has changed
-            instances_updated: usize,
-
-            /// number of instances whose VMM state has changed
-            vmms_updated: usize,
-
-            /// number of instances which the sled-agent indicated no longer exists
-            not_found: usize,
-
-            /// number of unexpected errors returned by sled-agent
-            sled_agent_errors: usize,
+            /// number of stale instance metrics that were deleted
+            pruned_instances: usize,
 
-            /// number of instances for which the sled agent was unreachable
-            unreachable_instances: usize,
+            /// instance states from completed checks
+            instance_states: BTreeMap<String, usize>,
```

Comment:
It's hard to tell from looking at this struct -- what are the String key values here, and in the other fields below?

Reply:
I think this is a mapping of "stringified state" -> "count of instances in that state"? This seems fine, but it might be nice to document that here. I guess that's why you implement "Display" below?

It might be nice to include some words of caution that "any additional context added to states/failures/checks, which causes the struct to print differently, will also be counted as a distinct {state,check,failure}"

Reply:
let me know what you think of the docs I've added in 70b8485b28a8b7047cdde8576bedf2afb8ea722b?

Reply:
Looks great, thank you!